content/desktop/use-desktop/builds.md

```diff
@@ -105,8 +105,17 @@ The **Configuration** section of the Info tab shows parameters passed to the bui
 ### Outputs and artifacts
 
 The **Build results** section shows a summary of the generated build artifacts,
-including the image manifest media type and digest, and information about build
-attestations such as SBOM and provenance data.
+including image manifest details, attestations, and Open Telemetry traces.
+
+Attestations are metadata records attached to a container image.
```

Comment:
minor nitpick because I ran into this when I tried to suppress that attestations are created with `--provenance=false`

```suggestion
Attestations are metadata records which can be attached to a container image.
```